Glossary_of_Terms.txt

```diff
@@ -34,6 +34,8 @@ RPG: Role-Playing Game, such as Dungeons and Dragons or Compound X!
 
 Player Character: The Player's avatar in the game world. A Player Character is made by following the procedures outlined in the CharacterCreation folder. Generally, a Player only controls one Character at a time; that Character is called a Player Character. 
 
+Proficiency: Compound X uses three types of Proficiencies Weapon Proficiency, Armor Proficiency, and Skill Proficiency. For Weapon or Armor Proficiency, see Basic Rules Assigning A Class To Your Character Section, subsections Weapon Proficiency or Armor Proficiency. For Skill Proficiency see Basic Rules RP/Skills section.  
```

Comment:
Can you add some text that describes what a proficiency is? Like: "Proficiency: sufficient aptitude such that a character can use a given type of Weapon/Armor or use a Skill with higher capability." 

>Compound X uses three types of Proficiencies Weapon Proficiency, Armor Proficiency, and Skill Proficiency

Can you add a colon after "Proficiencies" or something here? There is a distinct pause but no punctuation.
And I think we really need to use quotes or something to indicate that we are referring to a section or chapter. For example, see Basic Rules doc.

>For Skill Proficiency see Basic Rules RP/Skills section

I think we need a comma after "Proficiency," it helps readability because of the pause from the prepositional phrase. 

>see Basic Rules Assigning A Class To Your Character Section 

I think we need to get in the habit (myself included) of putting quotes around titles/sections or all caps or _anything_ so that it's easier to read. Example:
"... see Basic Rules "Assigning A Class To Your Character" section, subsections "Weapon Proficiency" or..."
or better yet, what I've been trying to do (due to the weird pauses in the quotation) is as follows:
"... see section in the Basic Rules "Assigning A Class to Your Character," subsections "Weapon Proficiency"..."

Reply:
wilco. And we should add something to that effect to the style guide, if we haven't already.

Reply:
Thanks for putting up with my nitpicks! :1st_place_medal: 
I added a PR for it